.github/workflows/cut_release_branch.yml

```diff
@@ -10,6 +10,7 @@ jobs:
       - uses: actions/checkout@v3
         with:
           fetch-depth: '0'
+          token: ${{ secrets.SERVICES_ACCESS_TOKEN }}
```

Comment:
Is the default token `GITHUB_TOKEN`, if we don't mention PAT?

Reply:
Yep. You can look at the checkout default over here
https://github.com/actions/checkout

```
    # Personal access token (PAT) used to fetch the repository. The PAT is configured
    # with the local git config, which enables your scripts to run authenticated git
    # commands. The post-job step removes the PAT.
    #
    # We recommend using a service account with the least permissions necessary. Also
    # when generating a new PAT, select the least scopes necessary.
    #
    # [Learn more about creating and using encrypted secrets](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/creating-and-using-encrypted-secrets)
    #
    # Default: ${{ github.token }}
    token: ''
```